### PR TITLE
Index file

### PR DESCRIPTION
**Pull request by rominana**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+        <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+       <div id="main">
+         <h1 id="title">
+          Albert Einstain
+         </h1>
+       <div id="img-div">
```
Comment (rominana) [inline index.html:15]: ### Semantic tags

- It would be better to use semantic tags in this case. Ex. `<main>`

**Review by rominana — COMMENTED** [review]
Hi @rominana! 
Good job so far! :fire:
There are some issues that you still need to work on to go to the next project but you are almost there! :rotating_light:

Let's take a look :eyes:

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @rominana,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ Polite comments
✔️ Easy to understand comments

## Required Changes ♻️

- [ ] You don't always include an explanation for each requested change, please explain why you suggested the changes

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rominana — COMMENTED** [review]
### Hi @user,
Good job so far 👍🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

**Changes Required** :recycle:

- [ ]  Please, add a title and description to your PR

- [ ]  Add a README.md file with the proper information about your project

- [ ]  Be sure to add your commits in the imperative form

- [ ]  Add linters for HTML and CSS so you can check your code

- [ ]  Remember to use the correct flow :

Please, check my comments below and make all required changes.

Submit for a re-review when you're done. :1st_place_medal: 

Thanks! :+1: :smile_cat:

**Review by Munsa1 — APPROVED** [review]
Hi @rominana , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Inline comments
:heavy_check_mark: respectful tone
:heavy_check_mark: Gve reason why

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**